### PR TITLE
Script for benchmarks on specific versions of artipie

### PR DESCRIPTION
**Pull request by Sammers21**
Related to #21 

This script would allow gathering data for the already released version of artipie.

### CODE REVIEW

**Comment by Sammers21** [comment]
@olegmoz, review the PR, please.

File: specific-versions.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Enter script located dir, whaterver the script is called from
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+if [ -z ${TF_VAR_secret_key+x} ]; then echo "TF_VAR_secret_key is unset"; exit 1; else echo "TF_VAR_secret_key is set to '$TF_VAR_secret_key'"; fi
+if [ -z ${TF_VAR_access_key+x} ]; then echo "TF_VAR_access_key is unset"; exit 1; else echo "TF_VAR_access_key is set to '$TF_VAR_access_key'"; fi
```
Comment (olegmoz) [inline specific-versions.sh:7]: @Sammers21 this check is duplicated without good reason

**Review by olegmoz — CHANGES_REQUESTED** [review]
@Sammers21 please see the comment

File: specific-versions.sh (hunk 1 above)
Comment (Sammers21) [reply specific-versions.sh:7]: @olegmoz, no duplications `TF_VAR_secret_key `and `TF_VAR_access_key` are different variables

Comment (olegmoz) [reply specific-versions.sh:7]: @Sammers21 oh, sorry, my bad

**Review by olegmoz — APPROVED** [review]

**Pull request merged**